baremetal_network_provisioning/common/validators.py

```diff
@@ -57,6 +57,18 @@ def validate_request(request):
     return body
 
 
+def validate_switch_attributes(keys, attr_keys):
+    """Validate the keys in request body."""
+    extra_keys = set(keys) - set(attr_keys)
+    if extra_keys:
+        msg = _("Unrecognized attribute(s) '%s'") % ', '.join(extra_keys)
+        for extra_key in extra_keys:
+            if extra_key == 'family':
+                return
```

Comment:
Line 67, instead of return do continue for valid keys so that invalid keys will not be accepted.